cluster/addons.go

```diff
@@ -313,6 +339,20 @@ func (c *Cluster) deployCoreDNS(ctx context.Context, data map[string]interface{}
 	return nil
 }
 
+func getDNSAutoscalerLinearModeParams(LinearAutoscalerParams *v3.LinearAutoscalerParams) (string, error) {
+	// The dns autoscaler uses linear mode of cluster-proportional-autoscaler to determine number of replicas
```

Comment:
would it be possible to define this logic in k-d-m template? Something like: `if !params then respect linearmode`